frontend/src/components/associations/settings/AssociationNamespaceSettings.tsx

```diff
@@ -0,0 +1,232 @@
+                            {errors.namespaceName && touched.namespaceName && (
+                                <Form.Control.Feedback type={"invalid"}>
+                                    {errors.namespaceName}
+                                </Form.Control.Feedback>
+                            )}
+                        </InputGroup>
+                    </form>
+                )}
+            </Formik>
+        </>
+    );
+};
+
+const ListNamespaceTags = ({ namespace, setSelectedNamespace }) => {
+    return (
+        <Col md={"4"} xs={"6"}>
+            <Card>
+                <Card.Body>
+                    <span className={"float-right"}>
+                        <Button
+                            size={"sm"}
+                            className={"mr-2"}
+                            onClick={() => setSelectedNamespace(namespace)}
+                        >
+                            Détails
+                        </Button>
+                    </span>
+
+                    <Card.Title>{namespace.name}</Card.Title>
+                </Card.Body>
+            </Card>
+        </Col>
+    );
+};
+
+const NamespaceTagsModal = ({ namespace, closeModal }) => {
+    const newToast = useContext(ToastContext);
+
+    const { data, status, error } = useBetterQuery<
+        PaginatedResponse<TagModel[]>
+    >(
+        ["namespace.tags.list", { namespace: namespace.id, page_size: 1000 }],
+        api.tags.list
+    );
+
+    const [mutate] = useMutation(api.namespaces.delete, {
+        onSuccess: (response) => {
+            queryCache.refetchQueries("association.namespaces.list");
+            newToast({
+                message: "Namespace supprimé",
+                level: ToastLevel.Success,
+            });
+        },
+        onError: (errorAsUnknown) => {
+            const error = errorAsUnknown as AxiosError;
+            newToast({
+                message: `Erreur. Merci de réessayer ou de contacter les administrateurs si cela persiste. ${error.message}`,
+                level: ToastLevel.Error,
+            });
+        },
+    });
+
+    let type = tablerColors[hashCode(namespace.name) % tablerColors.length];
+
+    const deleteNamespace = () => {
+        mutate(namespace.id).then((_) => closeModal());
+    };
+
+    return (
+        <>
+            {status === "loading" ? (
+                <Loading />
+            ) : status === "error" ? (
+                <Error detail={error} />
+            ) : (
+                <Modal show={true} onHide={closeModal}>
+                    <Modal.Header closeButton>
+                        <Modal.Title>Tags dans {namespace.name} </Modal.Title>
+                    </Modal.Header>
+                    <Modal.Body>
+                        {(data?.results.length || 0) > 0 ? (
+                            data?.results.map((tag) => (
+                                <span key={tag.id}>
+                                    <Tag type={type} tag={tag.value} />
```

Comment:
Never paid attention to this before, but I feel like `type` should be called `color`.